src/main/kotlin/io/easybreezy/user/api/Router.kt

```diff
@@ -49,14 +53,15 @@ class Router @Inject constructor(
             post<Response.Either<Response.Ok, Response.Errors>, Create>("") { command ->
                 controller<UserController>(this).create(command)
             }
-
-            post<Response.Either<Response.Ok, Response.Errors>, UpdateContacts>("/update-contacts") { command ->
-                controller<UserController>(this).updateContacts(command, resolvePrincipal<UserPrincipal>())
-            }
             post<Response.Either<Response.Ok, Response.Errors>, UpdateActivities, ID>("/{id}/activities") { command, params ->
                 controller<UserController>(this).updateActivities(params.id, command)
             }
         }
+
+        post<Response.Either<Response.Ok, Response.Errors>, UpdateContacts>("/update-contacts") { command ->
+            controller<UserController>(this).updateContacts(command, resolvePrincipal<UserPrincipal>())
+        }
+        get<Response.Data<User>>("/me") { controller<UserController>(this).me(resolvePrincipal<UserPrincipal>()) }
```

Comment:
We should check that this user has access.

Reply:
Isn't user already being verified by principals? 
Otherwise are we need activity for default user info? As i remember It is used by frontend to check auth.

Reply:
yep, my fault